chat-components/src/components/chatbutton/ChatButton.tsx

```diff
@@ -19,15 +19,26 @@ function NotificationBubble(props: IChatButtonProps, parentId: string) {
         root: Object.assign({}, defaultChatButtonNotificationBubbleStyles, props.styleProps?.notificationBubbleStyleProps)
     };
 
+    const ihiddenTextStyles: React.CSSProperties = {
```

Comment:
We are reverting the component changes after the other PR checks in right?

Reply:
yes, this won't show once the components PR is in.